cedar-lean/Cedar/Thm/Validation/Validator.lean

```diff
@@ -36,7 +36,7 @@ def AllEvaluateToBool (policies : Policies) (request : Request) (entities : Enti
   ∀ policy ∈ policies, EvaluatesToBool policy.toExpr request entities
 
 def RequestAndEntitiesMatchSchema (schema : Schema) (request : Request) (entities : Entities) :Prop :=
-  ∀ env ∈ schema.toEnvironments,
+  ∃ env ∈ schema.toEnvironments,
```

Comment:
Maybe add a doc comment on this predicate explaining what it does and why you want "exists" instead of "forall". (I imagine "forall" is more intuitive, which is why we used it originally.)

Reply:
It was actually me that wrote that originally and I just understood it very wrong, and no one caught it in the last PR. I can add a doc comment about it though.